js/private/js_library.bzl

```diff
@@ -219,6 +245,12 @@ def _js_library_impl(ctx):
         sources = transitive_sources,
         data = ctx.attr.data,
         deps = ctx.attr.srcs + ctx.attr.declarations + ctx.attr.deps,
+        files = ctx.files.data,
```

Comment:
These files being copied is what fixes the jest use case.

Reply:
Hmm. Lets call this `data_files` to be very clear that you have to pass both `ctx.attr.data` and `ctx.files.data`